src/other/Table.js

```diff
@@ -77,7 +110,9 @@
         return {
             _id: d,
             element: function () {
-                return context.tbody.selectAll("tr").filter(function (d2) { return d2 === d; });
+                return context.tbody.selectAll("tr").filter(function (d2) { 
+                    return d2 === d; 
+                });
             }
```

Comment:
I can return this to its original state.  I was originally putting additional code here but removed it.